pkg/api/v1beta1/clustercurator_types.go

```diff
@@ -56,6 +56,16 @@ type Hook struct {
 	// and is a known Ansible entity.
 	// +kubebuilder:pruning:PreserveUnknownFields
 	ExtraVars *runtime.RawExtension `json:"extra_vars,omitempty"`
+
+	// A comma-separated list of tags to specify which sets
```

Comment:
Just a question, what happens when names overlap??

Reply:
We will just generate the AnsibleJob with the overlapping names and pass the responsibility to the AnsibleJob operator/controller to handle it.